modules/installation-vsphere-machines.adoc

```diff
@@ -71,11 +64,14 @@ $ base64 -w0 <installation_directory>/worker.ign > <installation_directory>/work
 $ base64 -w0 <installation_directory>/append-bootstrap.ign > <installation_directory>/append-bootstrap.64
 ----
 
-. Obtain the {op-system} OVA image:
+. Obtain the {op-system} OVA image from the
+link:https://access.redhat.com/downloads[Product Downloads] page on the Red
+Hat customer portal or the
+link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.1/[{op-system} image mirror]
```

Comment:
I saw there is nothing for GA version yet, is there is some available preview and downloadable rhcos image?

Reply:
If the link only point user to rhcos parent folder, but not some specific version of folder, customer need navigate to proper folder manually, I am OKAY to merge it.